CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [4.0.0]
+
+ - Updated ComfortPay implementation according new WSDL defined by TB. Changed ChargeRequest parameters please see [readme.md](readme.md) to reflect actual parameters.  
```

Comment:
Please state all breaking changes here. Nobody who already implemented the gateway will be reading the whole README to find out what needs to be changed in their code.

Reply:
Tomaj actually asked only for this 19 days ago:

> Can you please add some simple .md file with info about upgrade from actual version 3.0.0 to new major version